includes/class-sensei.php

```diff
@@ -793,7 +800,9 @@ public function check_user_permissions( $page = '' ) {
 				$lesson_id        = get_post_meta( $post->ID, '_quiz_lesson', true );
 				$lesson_course_id = get_post_meta( $lesson_id, '_lesson_course', true );
 
-				$update_course = Sensei_WC::course_update( $lesson_course_id );
+				if ( method_exists( 'Sensei_WC', 'course_update' ) ) {
+					Sensei_WC::course_update( $post->ID );
```

Comment:
And here too?

Reply:
Good catch. 9ee50332b4693a93209ab726fbadb5ee3dcd146a